user_taste.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from twitch import TwitchClient
+
+
+def get_all_follows(user_id: str, show_progress=False):
+    """Return a list of all channels a user follows"""
```

Comment:
Can we have the same description format as the one below:
"""
Description
"""